docs/spfx/compatibility.md

```diff
@@ -0,0 +1,60 @@
+
+Because SharePoint Online and the on-premises versions of SharePoint Server have different release cycles for new capabilities, they also have support different Sharepoint Framework capabilities.
+
+SharePoint Online always uses the latest version of the SharePoint Framework, but SharePoint 2016 and SharePoint 2019 only support the versions that match the server-side dependencies of the deployed packages.
+
+|SharePoint version|Supported SPFx version|Supported features
+|---|---|---|
+|SharePoint Online|All versions|All features
+|SharePoint Server 2019|v1.4.1 or lower|SPFx client-side web parts in classic and modern pages, and extensions in modern pages.
+|SharePoint 2016 Feature Pack 2|v1.1|SPFx client-side web parts hosted in classic SharePoint pages.
+
+For more information about SharePoint Framework development with SharePoint 2016 Feature Pack 2 and SharePoint 2019, refer to the following:
+
+* [SharePoint Framework development with SharePoint 2016 Feature Pack 2](sharepoint-2016-support.md)
+* [SharePoint Framework development with SharePoint Server 2019](sharepoint-2019-support.md)
+
+## SPFx development environment compatibility
+
+As each new version of the SharePoint Framework is released, support for newer versions libraries is constantly added to ensure that the toolset remains up-to-date.
+
+The following table lists each version of SharePoint Framework along with each library versions are compatible:
+
+|SPFx|Node.js|NPM|TypeScript|React
+|---|---|---|---|---|---|
+|[1.10.0](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.10-release-notes)|LTS v10.x|v5|v3.3|v16.8.5|
+|[1.9.1](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.9.1-release-notes)|LTS v10.x| v5|v2.9|v16.8.5|
+|[1.9.0](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.9-release-notes) | LTS 8.x, LTS 10.x |v5 |v2.9|v16.8.5|
+|[1.8.2](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.8.2-release-notes)| LTS 8.x, LTS 10.x | v5 |v2.9|v16.7.0|
+|[1.8.1](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.8.1-release-notes)| LTS 8.x | v5|v2.7, v2.9, v3.x|v16.7.0|
+|[1.8.0](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.8-release-notes)| LTS 8.x | v5|v2.7, v2.9, v3.x|v16.7.0|
+|[1.7.1](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SPFx-Package-Version-1.7.1)| LTS 8.x | v5 |v2.4|v16.3.2|
+|[1.7.0](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.7-release-notes)| LTS 8.x | v5 |v2.4|v16.3.2|
+|[1.6.0](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.6-release-notes)|LTS 6.x,  LTS 8.x | v5|v2.4|v15.x|
+|[1.5.1](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SPFx-Package-Version-1.5.1)|LTS 6.x,  LTS 8.x | v5|v2.4|v15.x|
+|[1.5.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SharePoint-Framework-Package-v1.5)|LTS 6.x,  LTS 8.x | v3.x|v2.4|v15.x|
```

Comment:
The lowest version of npm included in Node v8 is npm v5, so perhaps it would be good to clarify that npm v3 applies to Node v6 and npm v5 to Node v8

Reply:
You're absolutely right. Added the recommended changes.